pkg/stanza/operator/transformer/add/add_test.go

```diff
@@ -297,6 +297,24 @@ func TestProcessAndBuild(t *testing.T) {
 			},
 			false,
 		},
+		{
+			"add_expr",
+			func() *Config {
+				cfg := NewConfig()
+				cfg.Field = entry.NewAttributeField("fookey")
+				cfg.Value = "EXPR('foo_' + body.key)"
```

Comment:
Works for me, but would be best if you use the env function there to prove the patcher is doing its job, wdyt?